### DISCUSSION TITLE
Question regarding Applying Model

### DISCUSSION THREAD

**Discussion by yedhu-artefact** [moj-analytical-services/splink#360] [Q&A] [answered]
Hi Team,

First of all thanks for creating such an amazing package!

I wanted to know how I can apply the model once trained, is there a way to do it? 

Currently, it seems to me that both the training and application of the model are clubbed together in the function get_scored_comparisons().

Thanks in advance!

**Comment by RobinL** [comment, marked as answer]
Yes - you can use https://github.com/moj-analytical-services/splink/blob/87a710886ab420e5254482905fea8938a392fab1/splink/__init__.py#L82

In the (forthcoming) Splink3, the training and prediction steps are separated out to make this simpler.  If you're just starting out with Splink, and particularly if you're working with smaller data (<1m rows), I'd recommend using Splink 3.  It's faster and simpler to use.  See the note at the top of the main Splink repo for more info.
